blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -177,6 +182,17 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
         }
     }
 
+    private String getResponseBody( List<String> errors ) {
+        StringBuilder sb = new StringBuilder();
+        sb.append( "The following errors have been encountered:" + System.lineSeparator() );
+
+        for( String error : errors ) {
+
+            sb.append( error + System.lineSeparator() );
+        }
+        return sb.toString();
```

Comment:
I figure that's a whole new can of worms.  I think we should do it in the future, but our other issues make it look like a can of small beans. :-P